pkg/api/register.go

```diff
@@ -118,52 +118,67 @@ func AddToScheme(scheme *runtime.Scheme) {
 func (obj *Pod) GetObjectMeta() meta.Object                                  { return &obj.ObjectMeta }
 func (obj *Pod) GetObjectKind() unversioned.ObjectKind                       { return &obj.TypeMeta }
 func (obj *PodList) GetObjectKind() unversioned.ObjectKind                   { return &obj.TypeMeta }
+func (obj *PodList) GetListMeta() meta.List                                  { return &obj.ListMeta }
```

Comment:
I swear this didn't work for one variant of the pattern.  Did you
verify that the non interface version was changed?